packages/ui/src/components/BlockList.tsx

```diff
@@ -59,8 +59,6 @@ class _BlockList extends React.Component<Props, {}> {
             : `Blocks from rank ${dag.minRank} to ${dag.maxRank}`
         }
         refresh={() => this.refresh()}
```

Comment:
You shouldn't pass in argument refresh if you want to hide the refresh button.

Reply:
Ok